src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -76,28 +77,44 @@ class Comment extends React.Component<Props, State> {
         isInputOpen: false,
     };
 
+    selectComment = (isSelected: boolean) => {
+        if (this.props.onCommentSelect) {
```

Comment:
Let's set a default prop value of `noop` to avoid this check, similar to the other callbacks.